src/api/index.js

```diff
@@ -185,9 +185,19 @@ export function getExternalLogURL({
   container,
   externalLogsURL,
   namespace,
-  podName
+  podName,
+  startTime,
+  completionTime
 }) {
-  return `${externalLogsURL}/${namespace}/${podName}/${container}`;
+  let queryStringParams = []
+  if (startTime) queryStringParams.push(`startTime=${startTime}`)
+  if (completionTime) queryStringParams.push(`completionTime=${completionTime}`)
+
+  let queryString = ""
+  if (queryStringParams.length) {
+    queryString = `?${queryStringParams.join('&')}`
+  }
```

Comment:
Can you update this to use `URLSearchParams` instead? This ensures that if in future we add values that need special encoding they'll be properly handled.

e.g.

```js
const queryParams = new URLSearchParams();
if (startTime) {
  queryParams.set('startTime', startTime);
}
if (completionTime) {
  queryParams.set('completionTime', completionTime);
}
let queryString = queryParams.toString(); // returns the properly encoded string, or '' if no params
if (queryString) {
  queryString = `?${queryString}`;
}

return `${externalLogsURL}/${namespace}/${podName}/${container}${queryString}`
```

Reply:
Great suggestion! Updated!